src/connectors/clear-refinements/connectClearRefinements.ts

```diff
@@ -94,7 +94,10 @@ const connectClearRefinements: ClearRefinementsConnector = function connectClear
       transformItems = (items => items) as TransformItems<string>,
     } = widgetParams || {};
 
-    if (widgetParams.includedAttributes && widgetParams.excludedAttributes) {
+    if (
+      (widgetParams || {}).includedAttributes &&
+      (widgetParams || {}).excludedAttributes
+    ) {
```

Comment:
```suggestion
    if (includedAttributes && excludedAttributes) {
```

why not like this?

Reply:
```
    const {
            includedAttributes = [],
            excludedAttributes = ['query'],
            ...
```

Yeah, it's because by default, they're filled with stuff.

Reply:
```suggestion
    if (
      widgetParams &&
      widgetParams.includedAttributes &&
      widgetParams.excludedAttributes
    ) {
```

This way is shorter :)